bpf/kmesh/cgroup_sock.c

```diff
@@ -40,29 +50,86 @@ struct {
 
 static const char kmesh_module_name[] = "kmesh_defer";
 
-static inline void record_netns_cookie(struct bpf_sock_addr *ctx)
+static inline void record_netns_cookie(struct bpf_map *map, struct bpf_sock_addr *ctx)
 {
     int err;
     int value = 0;
     __u64 cookie = bpf_get_netns_cookie(ctx);
-    err = bpf_map_update_elem(&map_of_manager, &cookie, &value, BPF_NOEXIST);
+    err = bpf_map_update_elem(map, &cookie, &value, BPF_NOEXIST);
     if (err)
         BPF_LOG(ERR, KMESH, "record netcookie failed!, err is %d\n", err);
 }
 
-static inline void remove_netns_cookie(struct bpf_sock_addr *ctx)
+void record_manager_netns_cookie(struct bpf_sock_addr *ctx)
+{
+    record_netns_cookie(&map_of_manager, ctx);
+}
+
+void set_netns_cookie_value(struct bpf_sock_addr *ctx, int value)
+{
+    __u64 cookie = bpf_get_netns_cookie(ctx);
+    int *old_value = bpf_map_lookup_elem(&map_of_manager, &cookie);
+    if (!old_value || *old_value == value)
```

Comment:
```suggestion
    if (!old_value && *old_value == value)
```

I think this should be and

Reply:
> I think this should be and

This function is used to enable the bypass function. The method is to modify the value of the record stored in the map. There are two cases in which the value does not need to be modified.
1. No data exists in the map.
2. The value stored in the map is the same as the input value (for example, multiple bypasses).